p3/plot/_cascade.py

```diff
@@ -33,12 +30,15 @@ def cascade(df, eff=None, size=(6, 5), **kwargs):
         "app" or "arch". If no value is provided, the efficiency is selected
         automatically based on the data available in `df`.
 
-    size: 2-tuple of floats, default: (6, 5)
+    size: 2-tuple of floats, optional
         The size of the plot, in backend-specific units.
+        In the matplotlib backend, the default is (6, 5), in the pgfplots
+        backend the size controls only the top left plot, where the default
+        is ("200pt", "200pt")
```

Comment:
Just a note for posterity: we should consider how to document backend-specific features like this.  If we add more backends, or more options with backend-specific features, then these lists could become very difficult to read.

Reply:
Agreed 👍 

I haven't made doc strings for the backends yet, because I wasn't quite clear on conventions.

However, I could see a path whereby this string just makes it clear that each backend sets a default and to refer to the individual backend documentation for default options.